lib/lambda_ethereum_consensus/fork_choice/handlers.ex

```diff
@@ -206,19 +200,15 @@ defmodule LambdaEthereumConsensus.ForkChoice.Handlers do
     # Make it a task so it runs concurrently with the state transition
     payload_verification_task =
       Task.async(fn ->
-        HardForkAliasInjection.on_deneb do
-          versioned_hashes =
-            block.body.blob_kzg_commitments
-            |> Enum.map(&Misc.kzg_commitment_to_versioned_hash/1)
-
-          %NewPayloadRequest{
-            execution_payload: payload,
-            parent_beacon_block_root: parent_beacon_block_root,
-            versioned_hashes: versioned_hashes
-          }
-        else
-          %NewPayloadRequest{execution_payload: payload}
-        end
+        versioned_hashes =
+          block.body.blob_kzg_commitments
```

Comment:
We should store this in a variable outside the closure, to reduce the amount of data we pass to the task.